### DISCUSSION TITLE
Error! Engines cannot retrieve results:  google ( HTTP connection error )

### DISCUSSION THREAD

**Discussion by Manifoldme** [searx/searx#3244] [Installation issue: Official wiki]
<html>
<body>
<h1 style="font-size: 2em; margin: 0.67em 0px; color: rgb(232, 230, 227); font-family: sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(24, 26, 27); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><a href="http://192.168.0.225:8888/stats" style="color: var(--darkreader-text--color-url-font); background-color: transparent; text-decoration: none;">Engine stats</a><span> </span>- google</h1>

Engine name | Scores | Result count | Response time | Reliability
-- | -- | -- | -- | --
google |   |   |   | 0

</div>
</body>
</html>

<html>
<body>
<h1 style="font-size: 2em; margin: 0.67em 0px; color: rgb(232, 230, 227); font-family: sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(24, 26, 27); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><a href="http://192.168.0.225:8888/stats" style="color: var(--darkreader-text--color-url-font); background-color: transparent; text-decoration: none;">Engine stats</a><span> </span>- bing</h1>

Engine name | Scores | Result count | Response time | Reliability
-- | -- | -- | -- | --
bing |   |   |   | 0

</div>
</body>
</html>

Any help would be much appreciated.[Engine stats](http://192.168.0.225:8888/stats) - bing
Engine name	Scores	Result count	Response time	Reliability
[bing](http://192.168.50.50:8888/stats?engine=bing)				0
Errors and exceptions
Exception	httpx.ConnectError	Percentage	100
Parameter	NoneNonewww.bing.com
Filename	searx/search/processors/online.py:96
Function	_send_http_request
Code	response = req(params['url'], **request_args)[Engine stats](http://192.168.0.225:8888/stats) - google
Engine name	Scores	Result count	Response time	Reliability
[google](http://192.168.0.225:8888/stats?engine=google)				0
Errors and exceptions
Exception	httpx.ConnectError	Percentage	100
Parameter	NoneNonewww.google.com
Filename	searx/search/processors/online.py:96
Function	_send_http_request
Code	response = req(params['url'], **request_args)

**Comment by unixfox** [comment]
Please try another working instance on https://searx.space

**Comment by Manifoldme** [comment]
> Please try another working instance on https://searx.space

I don't wanna use foreign Instance.

I installed Searx using original Wiki,
and changed IP in settings.yml 

I'm trying to figure out what's wrong with my set up,
every other engine works fine.

**Reply by unixfox** [reply]
Why you didn't specify that you were trying to install searx? There were no context in your initial message.

Maybe you could try SearXNG which seems to work fine with Google: https://docs.searxng.org/admin/installation.html and https://github.com/searxng/searxng/

**Comment by Manifoldme** [comment]
I already found a solution, the culprit was my DNS Proxy, I forgot to make an exception for the SearX instance.
Thanks
Sorry for double rising the issue, I thought closed threads will be not read anymore.
